### ISSUE TITLE
SIGSEGV on exit

### ISSUE THREAD

**Issue by OPNA2608** [Swartz27/xray-16#1586]
**Describe the bug**
```
Thread 1 "Primary thread" received signal SIGSEGV, Segmentation fault.
0x00007fffa44f2ad0 in ?? ()
(gdb) bt
#0  0x00007fffa44f2ad0 in ?? ()
#1  0x00007ffff7ef84d7 in ImFontAtlas::ClearInputData (this=this@entry=0x41ce10) at /build/source/Externals/imgui/imgui_draw.cpp:2095
#2  0x00007ffff7ef8779 in ImFontAtlas::Clear (this=this@entry=0x41ce10) at /build/source/Externals/imgui/imgui_draw.cpp:2134
#3  0x00007ffff7ef8799 in ImFontAtlas::~ImFontAtlas (this=<optimized out>, this=<optimized out>) at /build/source/Externals/imgui/imgui_draw.cpp:2086
#4  0x00007ffff7ea4a8a in IM_DELETE<ImFontAtlas> (p=0x41ce10) at /build/source/Externals/imgui/imgui.h:1962
#5  IM_DELETE<ImFontAtlas> (p=0x41ce10) at /build/source/Externals/imgui/imgui.h:1962
#6  ImGui::Shutdown () at /build/source/Externals/imgui/imgui.cpp:3693
#7  0x00007ffff7ea4ac5 in ImGui::DestroyContext (ctx=0x4169f0) at /build/source/Externals/imgui/imgui.cpp:3614
#8  0x00007ffff7e351ff in xray::editor::ide::~ide (this=<optimized out>, this=<optimized out>) at /build/source/src/xrEngine/editor_base.cpp:55
#9  0x00007ffff7e88ed7 in CRenderDevice::~CRenderDevice (this=<optimized out>, this=<optimized out>) at /build/source/src/xrEngine/device.h:31
#10 0x00007ffff7256e91 in __cxa_finalize () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6
#11 0x00007ffff7e2ee27 in __do_global_dtors_aux () from /nix/store/hgw8hjix0xzj5g0hldjpqpq09dnq7pv2-openxray-2188-unstable-2024-01-13/lib/xrEngine.so
#12 0x00007ffff7fc2460 in ?? ()
#13 0x00007ffff7fcb0e2 in _dl_call_fini () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2
#14 0x00007ffff7fced04 in _dl_fini () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2
#15 0x00007ffff7257426 in __run_exit_handlers () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6
#16 0x00007ffff725755e in exit () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6
#17 0x00007ffff723f0d5 in __libc_start_call_main () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6
#18 0x00007ffff723f189 in __libc_start_main_impl () from /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6
#19 0x0000000000401215 in _start ()
```

**To Reproduce**
1. `xr_3da`
2. Once in main menu, just "Exit to Windows"

**Expected behavior**
No segfault.

**Screenshots, videos**
n/a

**BugTrap error report**
n/a

**Desktop (please complete the following information):**
 - OS: NixOS 23.11
 - OpenXRay build version `2188-november-2023-rc1` tag as well as current master 692666fbab83bb0847bd6b16a43b6ceb8ebeb3d1

**Additional context**
Log, in case any details in there are relevant: [openxray.log](https://github.com/OpenXRay/xray-16/files/13932872/openxray.log)

**Comment by ericwomer** [comment]
Can confirm here. Frame 0 is ?? () even when built with debug info

OS: KUbuntu 23.10
OpenXRay Build: Dev [ebeb3d1](https://github.com/OpenXRay/xray-16/commit/692666fbab83bb0847bd6b16a43b6ceb8ebeb3d1)
Build: RelWithDebInfo

**Comment by Xottab-DUTY** [comment]
Should have been fixed in 1b121f18a8a5205c895a038e067abaf963141647. Assuming so and closing.
Please, reopen if crash persists!
